azkaban-web-server/src/main/java/azkaban/webapp/servlet/WebUtils.java

```diff
@@ -47,7 +47,7 @@
     try {
       azkabanEventReporter = ServiceProvider.SERVICE_PROVIDER
           .getInstance(AzkabanEventReporter.class);
-    } catch (NullPointerException | ConfigurationException e) {
+    } catch (Exception e) {
```

Comment:
We really, really want to catch _everything_ here. It is safe to leave `azkabanEventReporter` set to null.